### PR TITLE
Issue #668 Wrong URL of minishift start command reference in file "README.md".

### PR DESCRIPTION
**Pull request by warmchang**
Fix the https://github.com/minishift/minishift/issues/668

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ with the KVM hypervisor driver.
 
   - The IP is dynamically generated for each OpenShift cluster. To check the IP, run the `minishift ip` command.
   - By default Minishift uses the driver most relevant to the host OS. To use a different driver, set the `--vm-driver` flag in `minishift start`. For example, to use VirtualBox instead of KVM on GNU/Linux operating systems, run `minishift start --vm-driver=virtualbox`. For more information about `minishift start` options,
-  see the [minishift start command reference](https://minishift.io/docs/minishift_start.md).
+  see the [minishift start command reference](https://minishift.io/docs/minishift_start.html).
```
Comment (budhrg) [inline README.md]: 👍

**Comment by budhrg** [comment]
LGTM 👍

**Comment by Preeticp** [comment]
LGTM

**Comment by praveenkumar** [comment]
Thanks @warmchang for PR.

**Comment by hferentschik** [comment]
Guys, please check the commit message as well, when you look at a patch :-(

**Comment by praveenkumar** [comment]
@hferentschik we really missed it :(

**Pull request merged**